src/plugins/ui_actions/public/context_menu/build_eui_context_menu_panels.test.ts

```diff
@@ -36,46 +37,203 @@ const createTestAction = ({
     execute: async () => {},
   });
 
-test('contextMenu actions sorting: order, type, displayName', async () => {
+const resultMapper = (panel: EuiContextMenuPanelDescriptor) => ({
+  items: panel.items ? panel.items.map((item) => ({ name: item.name })) : [],
+});
+
+test('sorts items in DESC order by "order" field first, then by display name', async () => {
```

Comment:
Just curious - is there a particular reason why this needs to be sorted by display name? To me it feels like it doesn't add any additional value, just makes things a bit more complex and inconsistent between different languages.

Reply:
It needs to be sorted by something else other than the `order` field, the reason being that all actions of the same type have the same `order`. For example, if you create 3 Dashboard-to-Dashboard drilldowns it will attach 3 actions to a trigger and all of them will have the same `order`.  If we don't sort those by something else than `order` field, then effectively within that dashboard drilldown subgroup the same `order` items will appear in random order.